server/handlers.go

```diff
@@ -0,0 +1,283 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/big"
+	"net/http"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/golang/glog"
+	"github.com/livepeer/go-livepeer/common"
+	"github.com/livepeer/go-livepeer/eth"
+)
+
+func logAndRespondWithError(w http.ResponseWriter, errMsg string, code int) {
+	glog.Error(errMsg)
+	http.Error(w, errMsg, code)
+}
+
+func mustHaveFormParams(h http.Handler, params ...string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if err := r.ParseForm(); err != nil {
+			glog.Error(err)
```

Comment:
redundant since we're logging in the next line, right?

Reply:
oh, I see we're logging a different error here. why not wrap this error with our string and log just once with both data points?

Reply:
I notice this pattern below as well, so let's consider this thread relevant for all instances :)

Reply:
I wasn't sure if we wanted to return the same exact logged error message as a part of the HTTP response. However, after thinking about it, if an adversary is already able to send requests to these HTTP endpoints (i.e. via the CLI, curl, etc.) then returning informative HTTP errors is the least of the worries here...I think these HTTP endpoints are either secured against external parties by only accepting requests via localhost or they are not so, I think it makes sense to just wrap these errors so that they are returned in the HTTP response

Reply:
Yeah good thinking about potential attackers, but I also agree that it's probably wise for node operators to not expose this interface to the internet.

Reply:
Updated the handlers to just wrap any errors when creating the HTTP error response